jenkins/Jenkinsfile.release

```diff
@@ -91,7 +81,7 @@ pipeline {
                         sh 'rm -rf $GNUPGHOME'
                         sh 'gpg --import $GPG_FILE'
                         retry (3) {
-                            sh "bash $JENKINS_ROOT/deploy.sh true false"
+                            sh "bash $JENKINS_ROOT/deploy.sh true"
```

Comment:
I think we should keep "fasle", since deploy.sh needs this parameter to check databricks build or not.

Reply:
updated